src/components/forms/Datovelger.tsx

```diff
@@ -31,12 +31,12 @@ export function tilPeriode(gyldigTil: Date) {
 }
 
 function getAvgrensninger(): Avgrensninger {
-    const iDag = new Date();
+    const iDag = new Date(Date.now());
 
-    let iMorgen = new Date();
+    let iMorgen = new Date(iDag);
```

Comment:
`const` her du bør bruke, young Danielwalker?

![](https://media.giphy.com/media/MZW5o8f5RaH0Q/giphy.gif)

Reply:
Heh. Ja kanskje det, blir vel ikke re-assignet, men driver og manipulerer på objektet, så tenkte `let` var riktig å bruke for å vise intensjonen 😕 ❓